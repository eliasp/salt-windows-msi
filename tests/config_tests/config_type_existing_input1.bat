msiexec /i test.msi START_MINION="" /l*v config_type_existing_input1.install.log /qb
