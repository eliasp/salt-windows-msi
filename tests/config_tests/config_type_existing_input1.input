# User config line 1/3
master: master.user.config1
# User config line 2/3
id: id.user.config1
# User config line 3/3
