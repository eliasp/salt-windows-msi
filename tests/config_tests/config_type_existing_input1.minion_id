id.from.minion_id