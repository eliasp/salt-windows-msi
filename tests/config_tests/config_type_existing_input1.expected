# User config line 1/3
master: master.user.config1
# User config line 2/3
id: id.from.minion_id
# User config line 3/3
